Doc/library/exceptions.rst

```diff
@@ -965,9 +965,10 @@ their subgroups based on the types of the contained exceptions.
          def derive(self, excs):
             return Errors(excs, self.exit_code)
 
-   Like :exc:`ExceptionGroup`, any subclass of :exc:`BaseExceptionGroup` which
-   is also a subclass of :exc:`Exception` can only wrap instances of
-   :exc:`Exception`.
+   Subclasses of :exc:`BaseExceptionGroup` cannot wrap :exc:`BaseException`\ s
+   if they extend :exc:`Exception`, as for :exc:`ExceptionGroup`.  However,
+   there is no further restriction: a :exc:`BaseExceptionGroup` subclass
+   which extends :exc:`KeyError` could wrap any :exc:`Exception`.
```

Comment:
I found the wording here a little tricky, e.g. it wasn't fully clear to me how to parse the "as for ExceptionGroup" 
```suggestion
   A subclass of :exc:`BaseExceptionGroup` that extends :exc:`Exception`
   cannot wrap :exc:`BaseException`\ s, as is the case with :exc:`ExceptionGroup`. 
   However, there is no further restriction: a :exc:`BaseExceptionGroup` subclass
   which extends :exc:`KeyError` could wrap any :exc:`Exception`.
```